src/status_im/ui/screens/desktop/main/chat/views.cljs

```diff
@@ -152,7 +184,9 @@
        (time/timestamp->time timestamp)]])
    [react/view {:style styles/not-first-in-group-wrapper}
     [photo-placeholder]
-    [message-without-timestamp text message]]])
+    [react/view
+     [message-without-timestamp text message]
+     [link-preview text]]]])
```

Comment:
Wondering if it makes sense to only include `link-preview` if relevant text contains links. Because otherwise  `(.getPreview link-preview)` is invoked for each message, even if it's just text. See https://github.com/status-im/status-react/blob/8e7526e68db49355c89c72d974d1cb3f77e741de/src/status_im/chat/models/message_content.cljs#L83, you can check if `(-> message :content :metadata)` map contains a `:link` key.